scripts/prepare_automatic_backups.py

```diff
@@ -80,21 +95,33 @@ def update_cron_dict(cron_dict):
 
 
 def get_cron_dict():
+    """Converts yaml file content from yaml to dict type.
```

Comment:
The docstring makes it sound like this is an arbitrary yaml file but it looks like you're converting a specific yaml file; should be clearer about that. (I was surprised not to see an input argument.)

Reply:
done